### DISCUSSION TITLE
Downloads OS and mounts but nothing happens

### DISCUSSION THREAD

**Discussion by cowiea310** [grahampugh/erase-install#312] [Q&A] [answered]
Right now I am only looking to leverage the script to run macOS minor and major updates. I am not looking to erase and install any computers.
I followed the Use in JAMF Pro instructions and I am getting it to download a macOS 13 installer but it doesn't seem to install. I am sure I am missing something as my scripting ability is not at all where it should be. Any help would be appreciated.

````
   [erase-install] Caffeinating this script (pid=1265)
   [erase-install] Looking for existing installer app or pkg
   [find_existing_installer] Installer sparse image found at /Library/Management/erase-install/Install_macOS_13.0-22A380.sparseimage.
   [check_installer_is_valid] Checking validity of /Volumes/Install_macOS_13.0-22A380/Applications/Install macOS Ventura.app.
   [check_installer_is_valid] Mounting /Volumes/Install_macOS_13.0-22A380/Applications/Install macOS Ventura.app/Contents/SharedSupport/SharedSupport.dmg
   [check_installer_is_valid] Using Build value from com_apple_MobileAsset_MacSoftwareUpdate.xml
Volume Shared Support on disk6s2 force-unmounted
   [compare_build_versions] Comparing (1) 21G115 with (2) 22A380
   [check_installer_is_valid] Installer: 22A380 >= System: 21G115 : valid build.
   [erase-install] Running on architecture arm64
   [erase-install] Installer is at: /Volumes/Install_macOS_13.0-22A380/Applications/Install macOS Ventura.app
   [erase-install] Closing jamfHelper download message (language=en)

   [erase-install] Mounted installer will be unmounted: /Volumes/Install_macOS_13.0-22A380/Applications/Install macOS Ventura.app
Volume Install_macOS_13.0-22A380 on disk4s2 force-unmounted
   [erase-install] Cleaning working directory '/Library/Management/erase-install/content'

   [erase-install] attempting to terminate the 'caffeinate' process - Termination message indicates success
/Library/Application Support/JAMF/tmp/macOS update script: line 1046:  1277 Terminated: 15          /usr/bin/caffeinate -dimsu -w $$



   [finish] Script exit code: 0
```
`

**Comment by grahampugh** [comment, marked as answer]
Did you include the --reinstall argument? This is required to perform the OS upgrade (it is reinstalling the OS - not wiping).

**Reply by cowiea310** [reply]
I did. I followed the instructions on the wiki.  I first tried option 2 for just uploading the script.  Then I also tried option 1 and deployed the package.  Neither seemed to work.

For parameters I tried all of these:
Parameter 4: --force-curl
Parameter 5: --reinstall

Parameter 4: --force-curl
Parameter 5: --reinstall
Parameter 6: --update

Parameter 4: --force-curl
Parameter 5: --reinstall
Parameter 6: --os=13

**Reply by grahampugh** [reply]
Are you putting these parameter values in the script or the policy? I saw someone else a few days who had mistakenly been adding the values into the script.

**Reply by cowiea310** [reply]
I was putting them in the script in Jamf.  Once I put them in the policy it worked as it should.  Thank you for your help.  After going back and reading your instructions I see where you had it written out.
